zookeeper-api/src/main/java/org/apache/helix/zookeeper/datamodel/serializer/ZNRecordSerializer.java

```diff
@@ -38,6 +39,17 @@
 public class ZNRecordSerializer implements ZkSerializer {
   private static Logger logger = LoggerFactory.getLogger(ZNRecordSerializer.class);
 
+  // Reads from system property and represents whether auto compression is enabled or not.
+  // If and only if this property is set to "true", auto compression is enabled.
+  private final boolean autoCompressionEnabled =
```

Comment:
Is default value true with this statement?

Reply:
We need migration steps. So the first step will be let the default value to be true.